_posts/2023-08-20-my-lfx-mentorship-experience-with-the-cncf-strimzi-community.md

```diff
@@ -0,0 +1,170 @@
+Of course, it was not perfect, and this is where discussions and critical thinking helped me.
+
+### The mentors feedback
+
+After submitting my proposal, I got feedback from my mentors, and they were very positive about it.
+They had a lot of comments asking about my approach, and discussing and resolving the conversions was, of course, part of my win-win.
+At some point, I thought I wouldn’t be selected because my initial approach was mentioning unnecessary things like a full MQTT broker.
+
+The feedback from my mentors allowed me to rethink and defend my approach, proving that I had a basic understanding and was keen to work on the project.
+
+## The great day
+
+After so many discussions on the right approach, feedback, etc., I was not sleeping waiting for my results. 
+My first result was for the CNCF- Landing Page Improvement, in which I got rejected :(. 
+
+I started running unfaithfully and waiting for the Strimzi result, but it was too late, so I fell asleep hahaha.
+Waking up too early, I opened the email, and the great news was there
+
+<!--Image here -->
+
+## The Mentorship journey
+
+Upon joining the Strimzi community, I was warmly welcomed by mentors who were experts in their fields and exceptionally supportive and approachable.
+With their sense of humor, they gave me comprehensive guidance on understanding the Strimzi project governance, setting up a repository for the project I worked on, etc.
+This initial interaction set the tone for a productive and collaborative mentorship journey.
+
+### The work undertaken
+
+The project I worked on was the MQTT-Kafka Bridge, which is a bridge that allows you to produce messages to Kafka topics using MQTT.
+As I said before, I had to improve my solution, which could have been more manageable with many unnecessary components.
+Still, with the guidance of mentors, I could break down the task into manageable components.
+The mentors' feedback and code reviews were instrumental in refining my work and aligning it with Strimzi’s high standards.
+
+This MQTT-Kafka Bridge will enable one-way communication and mapping between MQTT and Kafka topics.
+The transformed messages will simultaneously be delivered to the Kafka cluster by the Kafka producer.
+This mapping makes it possible to integrate MQTT-based devices into an Apache Kafka system with interoperability, flexibility, and scalability.
+It would enable us to take advantage of the benefits of both MQTT and Kafka in our data processing and IoT system scenarios.
+
+This is what I worked on during each week.
+
+<!-- Image here -->
+
+The Bridge relies on a Mapper component to map the incoming MQTT messages to a valid Kafka Record, providing:
+- `interoperability`, e.g., MQTT QoS and Kafka Producer Acks level, MQTT wildcards topics, 
+- `flexibility`, the user sets his topic mapping rules
+- and `scalability`, the bridge supports n:m topic mapping.
```

Comment:
I am not sure I see this as "scalability", it's just about n to m mapping.

Reply:
@ppatierno fixed! :-)